client/Components/GameBoard/PlayerStats.jsx

```diff
@@ -145,19 +144,17 @@ const PlayerStats = ({
     };
 
     const getTide = () => {
-        if (stats && stats.tide && tideRequired) {
-            return (
-                <div className='state'>
-                    <img
-                        key='tide'
-                        onClick={onClickTide}
-                        className='img-fluid tide'
-                        src={Constants.TideImages[stats.tide]}
-                        title={t(`${stats.tide}-tide`)}
-                    />
-                </div>
-            );
-        }
+        return (
```

Comment:
Should I still check for stats here?

Reply:
Not sure why the check was there before, it should always be set. Guess we'll find out pretty quick after this goes live if it was needed..